hs_tools_resource/utils.py

```diff
@@ -45,7 +45,7 @@ def parse_app_url_template(url_template_string, term_dict_list=()):
 def get_SupportedResTypes_choices():
```

Comment:
This always returns CompositeResource and CollectionResource.  No reason to do introspection, there are no plans to support other resource types.

Is there a web app connector that uses Collection Resources?  If not, is that something we should consider removing and always assume we are working with composite resources?

Reply:
I am recalling a previous time I looked at the web app connector code and I think removing the resource type check (assuming connectors act on only composite) will speed up some of the logic around identifying web app connectors for a particular resource.

Reply:
It would get rid of the roundabout way resource types are solved.
https://github.com/hydroshare/hydroshare/blob/develop/hs_tools_resource/app_launch_helper.py#L34-L36

Reply:
> Is there a web app connector that uses Collection Resources? If not, is that something we should consider removing and always assume we are working with composite resources?

Here is a web app that uses Collection Resource: https://www.hydroshare.org/resource/bd52510b8b55413e85d6564756de7bea/

There may be more.

Reply:
This is the list of web apps that use CollectionResource.  I queried beta.  We can table removing CollectionResource as a supported resource type for web apps.  This method can still be updated to simply return Composite and collection resource types.

```'e682a66b9b804eda81f3c27c42d5d6ce'
'518a3a537a6244ee8fdf22ef494aab68'
'33c45575b95f45c4bf7335faab67ed36'
'3fe37ab4d74a448f90ed526c1a3a61ae'
'669262d53414424b94b54ac659b5faf6'
'7d472293c09a46c59cdef9160665603a'
'7efd9271134b43f6a4230914cf91b92d'
'1d7a9f179a5f48779d002e815bfc6a01'
'c477900488744e4a8e1df21326e4789b'
'96d1bc40a540434b80fc1df989faf8e2'
'abe59826a7af46e1b9c54222dc42fc77'
'9005b49f1c6140ae933516e9bc40581d'
'a0f43586759e462e9956a2e0361fc887'
'6d7ef1127ad04f019fe11714daf47e02'
'032f436b0145415c995ae94eb1d5ce28'
'e9686eadd4474b6587d83d9330d25854'
'4cfd280e8eb747169b293aec2862d4f5'
'66a4f9a5bac1463da01afc15edeacd9c'
'bd52510b8b55413e85d6564756de7bea'```